kratos/utilities/parallel_utilities.h

```diff
@@ -9,6 +9,7 @@
 //
 //  Main authors:    Riccardo Rossi
 //                   Denis Demidov
+//                   Philipp Bucher
```

Comment:
is that ok @RiccardoRossi ?

Reply:
more than fair to me!

@ddemidov  is it ok to you too?